### PR TITLE
Convert hocr-eval to use lxml, #8, add help #13

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: hocr-eval [hunk 1]
```diff
@@ -132,18 +141,12 @@ if(imgfile):
...
-actual_doc = ocrfile.read()
-ocrfile.close()
-
-# parse pages using beautiful soup
-gtsoup = BeautifulSoup(truth_doc,convertEntities='html')
-truth_pages = gtsoup('div','ocr_page')
-ocrsoup = BeautifulSoup(actual_doc,convertEntities='html')
-actual_pages = ocrsoup('div','ocr_page')
+truth_doc = html.fromstring(args[0])
+actual_doc = html.fromstring(args[0])
```
Comment (zuphilip) [inline hocr-eval]: The lines about reading the files with a special encoding where producing errors from `html.fromstring()`. Therefore I simply delete these two lines. Actually, I assume that normally the files are `UTF-8` encoded or state explicitely their encoding. Thus, I think it should be okay as it is now.

**Comment by zuphilip** [comment]
@kba Feel free to look through these changes here and comment.

File: hocr-eval (hunk 1 above)
Comment (kba) [inline hocr-eval]: Aren't you using the same file for `truth_doc` and `actual_doc`?

Comment (zuphilip) [inline hocr-eval]: Yes, indeed. Thank you for correcting the mistake. This should be ready now, right?

Comment (kba) [inline hocr-eval]: Looks good, yes. https://travis-ci.org/kba/hocr-tools/builds/131900845 Only failing test for me is hocr-dc-merge.

**Comment by zuphilip** [comment]
Thank you @kba!

**Pull request merged**